Xamarin.Forms.Controls/CoreGallery.cs

```diff
@@ -409,6 +409,7 @@ public override string ToString()
 				new GalleryPageFactory(() => new ViewCellGallery(), "ViewCell Gallery - Legacy"),
 				new GalleryPageFactory(() => new WebViewGallery(), "WebView Gallery - Legacy"),
 				new GalleryPageFactory(() => new BindableLayoutGalleryPage(), "BindableLayout Gallery - Legacy"),
+				new GalleryPageFactory(() => new DeviceTryOpenUriPage(), "Device TryOpenUri - Legacy"),
```

Comment:
I am not sure the new method is Legacy

Reply:
It's not about the method being a legacy.